src/views/AdminPage.vue

```diff
@@ -0,0 +1,153 @@
+<template>
+  <div>
+    <HeaderComp>
+      <div class="options" :style="style">Logo</div>
+    </HeaderComp>
+
+    <div class="container">
+      <SidebarComp />
+      <v-simple-table class="table" height="90vh">
+        <template v-slot:default>
+          <thead>
+            <tr>
+              <th class="text-left">Name</th>
+              <th class="text-left">Available</th>
+              <th v-show="user.role === 'admin'" class="text-left">Actions</th>
+              <th class="text-left">
+                <router-link to="/add">
+                  <v-btn v-show="user.role === 'admin'" color="primary">
+                    Add New Book
+                  </v-btn>
+                </router-link>
+              </th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr
+              :title="book.available ? '' : 'Out of stock'"
+              :class="book.available ? '' : 'disabled'"
+              v-for="book in books"
+              :key="book.id"
+            >
+              <td>
+                <v-img
+                  :src="book.image_url"
+                  class="align-end"
+                  gradient="to bottom, rgba(0,0,0,.1), rgba(0,0,0,.5)"
+                  height="100px"
+                  width="80px"
+                  cover
+                />
+                {{ book.title }}
+              </td>
+              <td>{{ book.available }}</td>
+              <td>
+
+                <v-btn
+                  v-show="user.role === 'admin'"
+                  @click="deleteBook(book.id)"
+                  color="primary"
+                  >Delete Book</v-btn
+                >
+              </td>
+            </tr>
+          </tbody>
+        </template>
+      </v-simple-table>
+    </div>
+    <alertDailog />
+  </div>
+</template>
+
+<script>
+import HeaderComp from "../components/HeaderComp.vue";
+import SidebarComp from "../components/SidebarComp";
+import alertDailog from "../components/Dailog";
+
+import axios from "axios";
+export default {
+  name: "AdminPage",
+  components: {
+    HeaderComp,
+    SidebarComp,
+    alertDailog,
+  },
+  data() {
+    return {
+      books: [],
+      user: {},
+      bookProp: [],
+      style: "text-indent: 4em;",
```

Comment:
style is a generic name. Also since try to put styles in class. Because in future if you need to add more then you can directly add there with more reability.